<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigMigrationWizard</class>
 <widget class="QWizard" name="ConfigMigrationWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration migration</string>
  </property>
  <property name="options">
   <set>QWizard::NoDefaultButton</set>
  </property>
  <widget class="QWizardPage" name="itemsPage">
   <property name="title">
    <string>Items to migrate</string>
   </property>
   <property name="subTitle">
    <string>This is a list of items found in the old configuration file, which can be migrated into the current configuration.</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeWidget" name="itemsTree">
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="optionsPage">
   <property name="title">
    <string>Options</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="dbGroup">
      <property name="title">
       <string>Put imported databases into separate group</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="groupNameEdit">
         <property name="placeholderText">
          <string>Group name</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>205</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
